src/Bindings/ManualBindings.cpp

```diff
@@ -1313,41 +1313,48 @@ static int tolua_SetObjectCallback(lua_State * tolua_S)
 
 static int tolua_cPluginLua_AddWebTab(lua_State * tolua_S)
 {
-	cPluginLua * self = reinterpret_cast<cPluginLua *>(tolua_tousertype(tolua_S, 1, nullptr));
+	cLuaState LuaState(tolua_S);
+	cPluginLua * self = nullptr;
 
-	tolua_Error tolua_err;
-	tolua_err.array = 0;
-	tolua_err.index = 3;
-	tolua_err.type = "function";
+	if (LuaState.GetStackValue(1, self))
```

Comment:
Use opposite condition and return out of the function, to avoid nesting too deep.